src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -150,18 +164,28 @@ public function execute(WorkflowInstance $workflow)
                 $email = Email::create();
                 try {
                     $email->setTo($member->Email);
-                } catch (Swift_RfcComplianceException $exception) {
+                } catch (RfcComplianceException) {
```

Comment:
This change fixes https://github.com/silverstripe/.github/issues/393  - though a separate PR will also target CMS 5 to fix it there as well.